lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -898,22 +899,22 @@ def test_cache_map_get_info_cmap(
             assert file_info == expected_msg
 
     @pytest.mark.parametrize(
-        "file_path, exp_file_info, exp_stream",
+        "file_path, exp_file_type, exp_stream",
         [
-            (Path(""), CacheType.DIRECTORY, "file_as_a_byte_stream"),
-            (Path("f1.json"), CacheType.FILE, "file_as_a_byte_stream"),
-            (Path("subdir1/subdir12"), CacheType.DIRECTORY, None),
+            (Path(""), CacheType.DIRECTORY, "<class '_io.BytesIO'>"),
+            (Path("f1.json"), CacheType.FILE, "<class '_io.BytesIO'>"),
+            (Path("subdir1/subdir12"), CacheType.DIRECTORY, "<class 'NoneType'>"),
         ],
```

Comment:
If you're going to go in this direction, it would be good to rename `exp_stream` to `exp_stream_type` or similar, since it contains a stringized type expression and not a stream.

But, I still recommend having `exp_stream` get set to `io.BytesIO(b"file_as_a_byte_stream")` and then having `mock_extract()` simply return `exp_stream`.